### ISSUE TITLE
docker-compose.yml version number invalid

### ISSUE THREAD

**Issue by scotstan** [microsoft/moabian#24]
Report from Muntazir Mehdi reporting:

```
ERROR: The Compose file './docker-compose.yml' is invalid because:

Invalid top-level property "iversion". Valid top-level sections for this Compose file are: version, services, networks, volumes, secrets, configs, and extensions starting with "x-".


You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.

For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/
```

## To track this down, I'll be asking for a screenshot of the following commands:

```
splash
docker-compose --version
docker --version
head -5 ~/moab/docker-compose.yml
```

My moab output:

![image](https://user-images.githubusercontent.com/58437613/107856629-515abb00-6dde-11eb-9cef-d337ef485f46.png)

**Comment by munti** [comment]
![screen-shot](https://user-images.githubusercontent.com/3047903/107866311-28f5af80-6e24-11eb-8b94-8135a601662e.jpg)

**Comment by munti** [comment]
· IP         192.168.1.21
 · Kernel     Linux 5.4.51-v7l+ armv7l
 · Load       0.04 0.01 0.00
 · CPU        59.9° C
 · URL        https://aka.ms/moab
 ○ Moabian    2.4.1

**Comment by scotstan** [comment]
You have the letter `i` as the first character in the file. That might be a `vim` side effect (insert mode). Maybe try deleting that? 

```
% vim moab/docker-compose.yml
x
[shift-ZZ]
% up
```

**Comment by munti** [comment]
I removed the letter "i". that was a good catch. I have a different problem, where when I try to do cd moab, cd moab
-bash: cd: moab: No such file or directory. Can you please post the link for a fully functional  docker-compose.yml file, so i could verify if it is correct.

**Comment by munti** [comment]
also, i noticed that the moab is not on the latest version - is there a way to update it.

**Comment by scotstan** [comment]
Once you log in via ssh:

```
cd moab
cat docker-compose.yml
```

If you don't have a moab folder, then ping me through email and we'll debug.

The `docker-compose.yml` file you need is here in the root of the repo:
https://github.com/microsoft/moabian/blob/main/docker-compose.yml

You have the only released & supported version, 2.4.1. We have a new branch v3 that we're testing this week for release ASAP. We'll be updating the readme, tags and main branch when released. There's a wiki for v3 migration, but I don't recommend it yet. 

Closing this particular issue as resolved (the 'i' issue).
